### ISSUE TITLE
FitBit CLI:  npx fitbit failing - "1/user/-/profile.json" missing

### ISSUE THREAD

**Issue by geoffhuntoon** [Fitbit/developer-bridge#158]
Hello! I'm using the FitBit CLI with `@fitbit/sdk` 4.1.0 and `@fitbit/sdk-cli` 1.7.2. When I run `npx fitbit`, I'm redirected to the browser (Chrome) as expected, but I consistently get the following error (non-relevant directory names removed):

```
>npx fitbit
No login information, starting login...
(node:15132) UnhandledPromiseRejectionWarning: Error: Fetch of 1/user/-/profile.json failed: no stored auth token
    at Object.apiFetch (...\node_modules\@fitbit\sdk-cli\lib\api\baseAPI.js:28:15)
    at async Object.userProfile [as default] (...\node_modules\@fitbit\sdk-cli\lib\api\userProfile.js:13:22)
    at async main (...\node_modules\@fitbit\sdk-cli\lib\cli.js:52:18)
(node:15132) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 1)
(node:15132) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
```

Once I close the browser the process terminates, and I'm unable to use the FitBit CLI at all. I haven't had any luck finding information regarding `1/user/-/profile.json` and the links to files in this stack trace haven't been very helpful. 

Thanks for taking a look!

**Comment by jtcooper10** [comment]
What shell are you using?

I was getting the same error consistently when attempting to login using Powershell. The issue magically went away when I used Git Bash instead.

**Comment by geoffhuntoon** [comment]
@jtcooper10 That's what I was worried about, but I had no workaround. If it still matters, I was using a Windows terminal accessed through IntelliJ. Thank you so much for the info, I'll try it out tonight!

**Comment by geoffhuntoon** [comment]
Update:  Trying the above has worked! This will make a good workaround for now. @Hexxeh (apologies if you're not the correct person to reach out to!) Should this issue be left open for investigation?

**Comment by Hexxeh** [comment]
Hi! This is an issue that's happened due to an breaking change within an upstream dependency package. We've investigated the issue and we'll be publishing a new version of @fitbit/sdk-cli that corrects the problem.

**Comment by Hexxeh** [comment]
Should be fixed via @fitbit/sdk-cli 1.7.3, please give it a try and re-open this ticket if you're still having issues after upgrading.
